### ISSUE TITLE
neofetch misbehaving

### ISSUE THREAD

**Issue by rugyada** [OpenMandrivaAssociation/distribution#2827]
**OpenMandriva version**:
Cooker build `#1317`

**Describe the bug**:

See screenshot

**Logs and screenshots if relevant**

![Screenshot_20220813_004640](https://user-images.githubusercontent.com/46781309/184454835-0a56a8c6-0347-4977-8950-27cba4694845.jpg)

**Comment by rugyada** [comment]
Possible cause

![Screenshot_20220821_150603](https://user-images.githubusercontent.com/46781309/185792524-cb854c77-2311-4fa5-9caa-e39aa31457f8.jpg)

**Comment by AngryPenguinPL** [comment]
Tbh nothing change at neofetch side. Maybe some changes related to shell?

**Comment by edwardp3** [comment]
Every neofetch screenshot I've found online, displays the color palette. Whatever @rugyada changed which resulted in the above screenshot, should fix it.

**Comment by edwardp3** [comment]
`neofetch --color-blocks on` should force it to display the color blocks, in Cooker it's not doing that.

![neofetch](https://user-images.githubusercontent.com/73129450/185812368-5603989a-d511-46e0-8669-b8d1f0162033.png)

**Comment by rugyada** [comment]
> One piece of info I found indicated that /etc/neofetch/config is the config file for it, but in Cooker, /etc/neofetch doesn't exist

I have a config file here
`~/.config/neofetch/config.conf`

**Comment by edwardp3** [comment]
I have that too. Didn't know that it was also in ~/.config. This is my color blocks section of that config file. Color blocks is on by default, but they're clearly not displaying in the terminal. I'm going to try to downgrade bash to see if the issue persists. 


```
# Color Blocks


# Color block range
# The range of colors to print.
#
# Default:  '0', '15'
# Values:   'num'
# Flag:     --block_range
#
# Example:
#
# Display colors 0-7 in the blocks.  (8 colors)
# neofetch --block_range 0 7
#
# Display colors 0-15 in the blocks. (16 colors)
# neofetch --block_range 0 15
block_range=(0 15)

# Toggle color blocks
#
# Default:  'on'
# Values:   'on', 'off'
# Flag:     --color_blocks
color_blocks="on"

# Color block width in spaces
#
# Default:  '3'
# Values:   'num'
# Flag:     --block_width
block_width=3

# Color block height in lines
#
# Default:  '1'
# Values:   'num'
# Flag:     --block_height
block_height=1

```

**Comment by edwardp3** [comment]
Installed `kuser` on my Plasma/Cooker VBox, changed the shell to /bin/sh and it had no affect on the neofetch display. Screenshot is from host OS.

![neofetch2](https://user-images.githubusercontent.com/73129450/185815472-cb542259-4f25-4a2b-95c0-22b10ad76827.png)

**Comment by rugyada** [comment]
Solved in neofetch-7.1.0.1

**Comment by edwardp3** [comment]
Neofetch updated to 7.1.0.1. Same output.

```
~]$ dnf info neofetch
Last metadata expiration check: 0:00:11 ago on Fri 26 Aug 2022 05:32:05 PM EDT.
Installed Packages
Name         : neofetch
Version      : 7.1.0.1
Release      : 1
Architecture : noarch
Size         : 342 k
Source       : neofetch-7.1.0.1-1.src.rpm
Repository   : @System
From repo    : cooker-x86_64
Summary      : Neofetch is a CLI system information tool written in BASH
URL          : https://github.com/dylanaraps/neofetch
License      : MIT
Description  : Neofetch is a CLI system information tool written in BASH.
             : Neofetch displays information about your system next to an
             : image, your OS logo, or any ascii file of your choice.

```
![neofetch](https://user-images.githubusercontent.com/73129450/186993799-10bb32d9-07a9-4f14-b6ed-88d05372a8fb.png)
